config/lua.txt

```diff
@@ -0,0 +1,7 @@
+## This file allows server hosts to specify the paths that the lua function "require" searches for modules in.
+## The character '?' will be replaced with the name of the module being searched.
+
+LUA_PATH ?
+LUA_PATH ?.lua
+LUA_PATH lua/?
+LUA_PATH lua/?.lua
```

Comment:
Add in `.luau` to both of these as well. Luau has its own syntax.

Eventually we'd want our own stuff to be on `.luau` as well, but GitHub doesn't highlight it yet.